src/constants/formatters.js

```diff
@@ -0,0 +1,9 @@
+import * as d3Format from 'd3-format';
```

Comment:
The thought was to make this a constant that can be used externally, much like `chartConstants`.